tests/scripts/helpers/list_metadata_json_verbose_true_schema.py

```diff
@@ -64,7 +64,7 @@
                                                                             }
                                                                         }
                                                                     },
-                                                                    "required": ["workload_name", "workload_type", "containers"]
+                                                                    "required": ["workload_name", "workload_type"]
```

Comment:
Why is containers removed?

Reply:
Below assertion error is encountered in cases where there are no containers for a job workload type as "containers" is a required field: 

> assert [<ValidationError: "'containers' is a required property">, <ValidationError: "'containers' is a required property">, <ValidationError: ''>] == ''

Examples:

```
"collect-profiles-28807155": {
                  "workload_name": "collect-profiles-28807155",
                  "workload_type": "job"
                },
"image-pruner-28802880": {
                  "workload_name": "image-pruner-28802880",
                  "workload_type": "job"
                }
```

Failed test logs : https://privatebin.corp.redhat.com/?cd5f4646e2d1f0e3#41XkaVvktuBJXmQDm4F11WTvfPXq4Rh7DRN164LyGUhQ